OptimizelySDK/Optimizely/OptimizelyClient.swift

```diff
@@ -24,7 +24,7 @@ open class OptimizelyClient: NSObject {
     // MARK: - Properties
     
     var sdkKey: String
-    var config: ProjectConfig?
+    var config: AtomicProperty<ProjectConfig> = AtomicProperty<ProjectConfig>()
```

Comment:
It's a bit annoying to see "property" everywhere. Can we use a wrapper prop to clean this up:

var atomicConfig = AtomicProperty<ProjectConfig>()
var config: ProjectConfig? {
    get { return atomicConfig.property }
    set { atomicConfig.property = newValue }
}